bloom-housing/bloom
# fix: patches translations for preferences

## Description

This patches preferences for listings and applications. The application now fetches the translated preferences, so all parts of a preference can now be translated.

## How Can This Be Tested/Reviewed?
Go to a listing with an application and preferences, choose a different language and you should see all of the preference translated. Since I didn't touch the preference components themselves, keep in mind that this still uses translation strings in some parts. You can ensure that the preferences are coming through translated properly by checking the listing data in session.

You can grab the necessary secrets from prod in Heroku and drop them into your backend/core .env.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
